docs/basic/getting-started/forms-and-events.md

```diff
@@ -98,3 +98,33 @@ If you don't quite care about the type of the event, you can just use React.Synt
+| DragEvent        | Drag and drop interaction with a pointer device (e.g. mouse).                                                                 |
+| FocusEvent       | Event that occurs when elements gets or loses focus.                                                                          |
+| FormEvent        | Event that occurs whenever a form or form element gets/loses focus, a form element value is changed or the form is submitted. |
+| InvalidEvent     | Fired when validity restrictions of an input fails (e.g `<input type="number" max="10">` and someone would insert number 20). |
+| KeyboardEvent    | User interaction with the keyboard. Each event describes a single key interaction.                                            |
+| MouseEvent       | Events that occur due to the user interacting with a pointing device (e.g. mouse)                                             |
+| PointerEvent     | Events that occur due to user interaction with a variety pointing of devices such as mouse, pen/stylus, a touchscreen and which also supports multi-touch. Unless you develop for older browsers (IE10 or Safari 12), pointer events are recommended. Extends UIEvent.                                                                                                                       |
+| TouchEvent       | Events that occur due to the user interacting with a touch device. Extends UIEvent.                                                                                                                           |
+| TransitionEvent  |  CSS Transition. Not fully browser  supported. Extends UIEvent                                                                                                                             |
+| UIEvent          |  Base Event for Mouse, Touch and Pointer events.                                                                                                                             |
+| WheelEvent       | Scrolling on a mouse wheel or similar input device. (Note:  `wheel` event should not be confused with the `scroll` event)                                                                                                                              |
+| SyntheticEvent   |  The base event for all above events. Should be used when unsure about event type                                                                                                                           |
+
+> **InputEvent ?**
+> You probably noticed that there is no InputEvent. This is because it is not supported by Typescript as the event itself has no fully browser support and may behave differently in different browsers.
+> It's suggested to use `KeyboardEvent` instead
+> Sources:
+>
+> - https://github.com/microsoft/TypeScript/issues/29441
+> - https://developer.mozilla.org/en-US/docs/Web/API/InputEvent
+> - https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event
```

Comment:
```suggestion
<details>
<summary>

**What about `InputEvent`?**

</summary>

You've probably noticed that there is no `InputEvent`. This is because it is not supported by Typescript as the event itself has no fully browser support and may behave differently in different browsers. You can use `KeyboardEvent` instead.

Sources:

 - https://github.com/microsoft/TypeScript/issues/29441
 - https://developer.mozilla.org/en-US/docs/Web/API/InputEvent
 - https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event

</details>
```